lib/iris/tests/results/analysis/rotated_pole.0.rlat.json

```diff
@@ -1 +1 @@
-{"std": 3.7195861803241614, "min": -5.593200206756592, "max": 8.72130012512207, "shape": [93, 75], "masked": false, "mean": 1.6348200228305594}
\ No newline at end of file
+{"std": 3.7195863723754883, "min": -5.5932002067565918, "max": 8.7213001251220703, "shape": [93, 75], "masked": false, "mean": 1.6348202228546143}
```

Comment:
It might have been "nicer" here to reduce the tolerance on the test for when it happens again (!).
I see these values *only just fail* the test at a relative tolerance of 1e-7, as default for `assertArrayAllClose` (+hence `assertDataAlmostEqual`).
You could maybe add something like `, reltol=5.0e-6` to the 4 calls [here](https://github.com/SciTools/iris/blob/5a6fba0fd86f56f0df4f6d19d9286358c42c99ca/lib/iris/tests/test_analysis.py#L560)
You could do that instead, or as well if you prefer (changes are *very* small after all).

Just a suggestion -- skip if you don't want.